docs/templating.md

```diff
@@ -972,7 +973,69 @@ a b c d e f
 ### dump (object)
 
 Call `JSON.stringify` on an object and dump the result into the
-template. Useful for debugging: `{{ foo | dump }}`.
+template. Useful for debugging: `{{ items | dump }}`. 
+
+**Input**
+
+```jinja
+{% set items = [{ a : 1}, { b : false}] %}
```

Comment:
I think you want to illustrate `dump` works on different primitives, so I'd use `['a', 1, { b : false}]` in all examples in the docs. The only thing you'd change between examples is the `space` parameter (none, 2 spaces, 1 tab).